lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -47,7 +51,8 @@ def start
 
   def client
     @_es ||= begin
-      adapter_conf = lambda {|f| f.adapter :excon }
+      excon_options = { client_key: @client_key, client_cert: @client_cert, client_key_pass: @client_key_pass }
+      adapter_conf = lambda {|f| f.adapter :excon, excon_options }
```

Comment:
@jimmidyson The nice thing about this is you aren't required to use an encrypted key; only if you specify a value for client_key_pass will it try to decrypt the key.